pkgs/servers/sunshine/default.nix

```diff
@@ -23,31 +25,33 @@
 , libva
 , libvdpau
 , numactl
+, amf-headers
+, svt-av1
+, vulkan-loader
 , cudaSupport ? false
 , cudaPackages ? {}
 }:
 
 stdenv.mkDerivation rec {
   pname = "sunshine";
-  version = "0.16.0";
+  version = "0.18.4";
 
   src = fetchFromGitHub {
     owner = "LizardByte";
     repo = "Sunshine";
     rev = "v${version}";
-    sha256 = "sha256-o489IPza1iLoe74Onn2grP5oeNy0ZYdrvBoMEWlbwCE=";
+    sha256 = "sha256-nPUWBka/fl1oTB0vTv6qyL7EHh7ptFnxwfV/jYtloTc=";
     fetchSubmodules = true;
   };
 
-  # remove pre-built ffmpeg; use ffmpeg from nixpkgs
+  # remove pre-built ffmpeg; restore ability to find and use external ffmpeg
   patches = [ ./ffmpeg.diff ];
```

Comment:
Where is the patch from? How do we update or rebase it?

Reply:
Good point -- a large part of the patch was a file that was removed in an earlier version of the package that I needed to find `ffmpeg` from `nixpkgs`. Instead of including in a patch, I used `fetchurl` to grab the file directly and include in the source tree before building. The rest of the patch is custom for `nixpkgs` to remove use of the upstream pre-built `ffmpeg`.